src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/ArticleRepository.php

```diff
@@ -34,4 +38,26 @@ public function findAllArticles()
     {
         return $this->createQueryBuilder('o')->getQuery();
     }
+
+    public function getQueryForRouteArticles(string $identifier, array $order) : SqlQuery
```

Comment:
It's optional feature, we don't need to do this now for every method. Let's take it easy and do it one by one when we will touch that function. Option to return `null` or specified type will be in `7.1` - so we can't use it everywhere for now.